CRM/Mailing/BAO/Mailing.php

```diff
@@ -916,6 +950,10 @@ private function _getTokens($prop) {
    * @return void
    */
   public function getTestRecipients($testParams) {
+    $session = CRM_Core_Session::singleton();
+    $sender_id = $session->get('userID');
```

Comment:
Hey Yashi, I'd never heard this as one of our standards before so I took a look & couldn't find it - have you got a link? https://www.drupal.org/coding-standards#naming

As an aside CRM_Core_Session::singleton()->getLoggedInUserID(); reduces the above to one line